### ISSUE TITLE
Strange bevahiour during Search and Replace

### ISSUE THREAD

**Issue by SalCag** [otykier/TabularEditor#547]
Hello

I'm using Tabular Editor 2.11.6 on a Windows Server 2016 machine.

I'm experiencing a strange behaviour when I'm using Search and Replace in the Expression Editor:
I have to replace a table name on several measures:
1. I click on a Measure
2. I open Search and Replace and replace all occurrences of a string
3. I click on the next measure, with the Search and Replace window still open
4. Click on the first line of the measure and click on Next
5. The first occurrence of the string is found
6. When I click on Replace the string will be replaced, but moved to the bottom of the code and a "Not Found" message  window opens
7. After Undo this step and restart the search and replace (With the Search and Replace window still open) it works

I was able to reproduce this effect on a Power BI Desktop file.

Can you please check this?

Thank you
Regards
Salvatore

**Comment by otykier** [comment]
Thanks for reporting this.

As a workaround, until this bug is fixed, consider using a script. For example, the following script will replace all occurences of the string "BEFORE" with the string "AFTER" in the DAX expression of the currently selected measure:

```csharp
Selected.Measure.Expression = Selected.Measure.Expression.Replace("BEFORE", "AFTER");
```

If you select multiple measures, use a loop like so:

```csharp
foreach(var m in Selected.Measures)
    m.Expression = m.Expression.Replace("BEFORE", "AFTER");
```

...or to simply loop through all measures across all tables use:

```csharp
foreach(var m in Model.AllMeasures)
    m.Expression = m.Expression.Replace("BEFORE", "AFTER");
```

**Comment by otykier** [comment]
Fixed in [2.11.7](https://github.com/otykier/TabularEditor/releases/latest).

**Comment by SalCag** [comment]
Thank you for fixing this
